### PR TITLE
[Tankerkoenig] Documentation: Changed reference from Paper UI to Main…

### PR DESCRIPTION
**Pull request by JueBag**
Changed reference from Paper UI to Main UI.  

Signed-off-by: Jürgen Baginski <opus42@gmx.de>

### CODE REVIEW

**Comment by Confectrician** [comment]
Maybe you could also remove the oh1 wiki reference at the bottom. (About line 190.)
We should be able to detect holidays with ephemeris feature nowadays.

**Comment by hmerk** [comment]
@JueBag 
As the name MainUI might change, we should not use this term. Ui should be enough.

**Review by Confectrician — APPROVED** [review]
LGTM

**Review by hmerk — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.tankerkoenig/README.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ Restart your server
 
 Either the web-request to Tankerkönig returned a failure or no valid response was received (this could be caused by a banned API-key).
 In both cases the Webservice and the Station(s) go OFFLINE.
-If the Tankerkönig return indicates an error a descriptive message (in German) is added next to the OFFLINE which will be displayed on the Webservice and Station(s) pages on Paper UI.
+If the Tankerkönig return indicates an error a descriptive message (in German) is added next to the OFFLINE which will be displayed on the Webservice and Station(s) pages on UI. For further investigation the API Explorer can be used to show all data-filds of the things.
```
Comment (fwolter) [inline bundles/org.openhab.binding.tankerkoenig/README.md:171]:
```suggestion
If the Tankerkönig return indicates an error a descriptive message (in German) is added next to the OFFLINE which will be displayed on the Webservice and Station(s) pages on UI. For further investigation the API Explorer can be used to show all data-fields of the things.
```

**Review by fwolter — CHANGES_REQUESTED** [review]

**Comment by fwolter** [comment]
You filed your PR directly against the main branch in the upstream repository. The normal way would be to first fork the addons repository and then file the PR from your forked repository.

**Comment by Confectrician** [comment]
Unfortunately that's the way it gets created, when you have access and use the github web ui for editing.
I have those problems too, when i just want to fix some small readme things and lazily use the web editor.

**Comment by fwolter** [comment]
Does it have any impact when filing it directly?

**Comment by JueBag** [comment]
> You filed your PR directly against the main branch in the upstream repository.

?? IMHO I pushed against https://github.com/openhab/openhab-addons/tree/JueBag-patch-1

**Comment by hmerk** [comment]
Correctly you should have a fork of the repo, make your changes in that fork and then create a PR. This can also be done trough web interface.

**Comment by fwolter** [comment]
That was the impact I was retrying to remember. You created a branch on the official repository. This is bad...

**Review by fwolter — APPROVED** [review]

**Comment by fwolter** [comment]
I deleted your branch in the official repo, after merging.

**Comment by JueBag** [comment]
> Correctly you should have a fork of the repo, make your changes in that fork and then create a PR. This can also be done trough web interface.

I'm sorry for that. @hmerk told me before, however I obviously missunderstood this autocreated branch as an own fork.

**Pull request merged**